packages/backend/src/ee/services/EmbedService/EmbedService.ts

```diff
@@ -814,6 +819,39 @@ export class EmbedService extends BaseService {
         };
     }
 
+    async calculateTotalFromSavedChart(
+        embedToken: string,
+        projectUuid: string,
+        chartUuid: string,
```

Comment:
We need the tileUuid so we can enforce dashboard filters if filter interactivity is disabled. Check logic in `getChartAndResults`.